governance/parameter-docs/param-dsr.md

```diff
@@ -0,0 +1,52 @@
+# $HumanReadableName
+
+```
+Alias: NA
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs:https://docs.makerdao.com/smart-contract-modules/rates-module/pot-detailed-documentation
+```
+
+## Description
+* What is this parameter?
+The Dai Savings Rate (DSR) is a variable rate of accrual earned by locking Dai in the DSR smart contract. Dai holders can earn savings automatically and natively while retaining control of their Dai. The DSR smart contract has no withdrawal limits, deposit limits, or liquidity constraints.
```

Comment:
So these docs are targeted at governance rather than DAI Holders. Goal is to make them as easy to understand as possible. 

"The Dai Savings Rate (DSR) is a variable rate of accrual earned by locking Dai in the DSR smart contract." - Variable Rate of Accrual is perhaps not as clear as it could be.

"Dai holders can earn savings automatically and natively while retaining control of their Dai." - Be more specific? They are earning more DAI?

"The DSR smart contract has no withdrawal limits, deposit limits, or liquidity constraints." - Might not important enough to be in the first paragraph? Might just come under considerations at the end? Maybe cut down to 'There is no limit to how much DAI maybe be deposited in the DSR Smart Contract.'?